eq-author-api/scripts/createMigration.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs").promises;
+const chalk = require("chalk");
+
+if (process.argv.length < 3) {
+  /* eslint-disable-next-line no-console */
+  console.info(
+    chalk.green(
+      "Not enough arguments provided to createMigration. Maybe the migration name is missing? Please refer to README."
+    )
+  );
+  return;
```

Comment:
Can we use `process.exit(1)` so the terminal picks it up that the command failed